src/EnergyPlus/ReportSizingManager.cc

```diff
@@ -1299,6 +1301,8 @@ namespace ReportSizingManager {
 						} else if ( OASysEqSizing( CurOASysNum ).HeatingAirFlow ) {
 							// Parent object sets heating flow rate
 							AutosizeDes = OASysEqSizing ( CurOASysNum ).HeatingAirVolFlow;
+						} else if ( DataDesicRegCoil ) {
+							AutosizeDes = FinalSysSizing( CurSysNum ).DesMainVolFlow;
```

Comment:
A regeneration air heating coil in OA system should use the "DesOutAirVolFlow", instead.  Corrected.